CONTRIBUTING.md

```diff
@@ -41,6 +41,45 @@ python -m SimpleHTTPServer 8080 .
 # navigate to http://localhost:8080
 ```
 
+To make your PRs more readable, you can provide this checkbox list to your PR's description. Additionally, people who wants to join translating docs into your language can easily see what's already done.
```

Comment:
```suggestion
To make PRs more readable, please provide this checklist to your PR's description so that other contributors can easily see what's already done.
```